src/Maestro/Client/src/Build.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.DotNet.Maestro.Client.Models
+{
+    public partial class Build
+    {
+        [JsonIgnore]
+        public string Repository => GitHubRepository ?? AzureDevOpsRepository;
```

Comment:
Having three members with the word *Repository may be confusing: "Repository", "GitHubRepository", "AzureDevOpsRepository". I'd rather have a function or change the name and add a preferred, precedence, etc. noun

Reply:
Do you mean something like `GetRepository()`?

Reply:
I agree it's a bit more readable that this is not a DB field. Changed to `Get*()`